docs/getting_started/index.md

```diff
@@ -4,86 +4,21 @@
-
-```sh
-wagtail start mysite
-```
-
-This will create a new folder `mysite`, based on a template containing everything you need to get started.
-More information on that template is available in
-[the project template reference](/reference/project_template).
-
-Inside your `mysite` folder, run the setup steps necessary for any Django project:
-
-```sh
-pip install -r requirements.txt
-python manage.py migrate
-python manage.py createsuperuser
-python manage.py runserver
-```
-
-Your site is now accessible at `http://localhost:8000`, with the admin backend available at `http://localhost:8000/admin/`.
-
-This will set you up with a new stand-alone Wagtail project.
-If you'd like to add Wagtail to an existing Django project instead, see [Integrating Wagtail into a Django project](/getting_started/integrating_into_django).
-
-There are a few optional packages which are not installed by default but are recommended to improve performance or add features to Wagtail, including:
-
--   [Elasticsearch](/advanced_topics/performance)
--   [Feature Detection](image_feature_detection)
-
 ```{toctree}
 ---
 maxdepth: 1
 ---
 tutorial
+quick_install
 demo_site
 integrating_into_django
 the_zen_of_wagtail
 ```
 
-(common_installation_issues)=
+Wagtail CMS is an open-source content management system that is built on [Django](https://www.djangoproject.com/), a popular Python web framework. It has gained popularity among developers and content editors for its powerful features and intuitive interface, providing a streamlined editing experience. Wagtail offers a comprehensive toolkit for content creation and management, including a rich text editor with formatting options, image and document management, version control, and content scheduling.
 
-## Common issues
+Developers appreciate Wagtail's highly customizable and modular architecture, which includes built-in support for Django's app structure. This allows them to easily create and integrate custom functionality, making Wagtail suitable for projects of any size. Wagtail excels in handling complex content structures, offering features like hierarchical page organization, robust search capabilities, and content localization.
 
-### Python is not available in `path`
-
-```sh
-python
-> command not found: python
-```
-
-For detailed guidance, see this guide on [how to add Python to your path](https://realpython.com/add-python-to-path/).
-
-### python3 not available
-
-```sh
-python3 -m pip install --upgrade pip
-> command not found: python3
-```
+Wagtail stands out as the preferred choice for tens of thousands of organizations globally, including renowned names like Google, NASA, and the British NHS. It has proven scalability, capable of handling high volumes of traffic from millions of visitors every month. What sets Wagtail apart is its ability to extend beyond traditional content management, providing seamless integration with data tools and rich data visualizations.
```

Comment:
```suggestion
Wagtail stands out as the preferred choice for tens of thousands of organisations globally, including renowned names like Google, NASA and the British NHS. It has proven scalability, capable of handling high volumes of traffic from millions of visitors every month. What sets Wagtail apart is its ability to extend beyond traditional content management, providing seamless integration with data tools and rich data visualisations.
```

For now we mostly avoid Oxford commas.

Reply:
The Google developer documentation style guide strongly advises the use of [serial comma or an Oxford comma](https://developers.google.com/style/commas).

Reply:
Ok great. No push back from me - I think we should update our writing guides to reflect this preference though. 

We often get lots of PRs changing things either to add or remove this comma, I just want us to have a preference.

Reply:
We should add a note above the section on Latin phrases. 

https://docs.wagtail.org/en/stable/contributing/general_guidelines.html#latin-phrases-and-abbreviations

Reply:
@lb- @thibaudcolas @vossisboss I would like your insights in regards to writing and creating content indicating the change from British spelling style to American, as well as our choice of documentation writing style guide. In my opinion, this is more related to documentation guidelines and should be included in [Documentation guidelines] (https://docs.wagtail.org/en/stable/contributing/documentation_guidelines.html#) or [Writing documentation] (https://docs.wagtail.org/en/stable/contributing/documentation_modes.html).

The reason why I suggest these two sections is that I believe documentation contributors will naturally look at them before any other section regarding documentation guidelines.

Reply:
@lb- @thibaudcolas @vossisboss Never mind! I can see now that @thibaudcolas already created an issue regarding this. Happy to work on the issue after this pull request is approved and merged.